consul/tests/test_e2e.py

```diff
@@ -0,0 +1,54 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+import pytest
+
+from datadog_checks.consul import ConsulCheck
+
+from . import common
+
+pytestmark = pytest.mark.e2e
+
+
+def test_e2e(dd_agent_check, instance_single_node_install):
+    aggregator = dd_agent_check(instance_single_node_install, rate=True)
+
+    aggregator.assert_metric('consul.peers', count=2)
+    aggregator.assert_metric('consul.catalog.nodes_critical', count=2)
+    aggregator.assert_metric('consul.catalog.services_passing', count=6)
+    aggregator.assert_metric('consul.catalog.nodes_warning', count=2)
+    aggregator.assert_metric('consul.catalog.services_warning', count=6)
+    aggregator.assert_metric('consul.catalog.services_critical', count=6)
+    aggregator.assert_metric('consul.catalog.services_up', count=6)
+    aggregator.assert_metric('consul.catalog.nodes_passing', count=2)
+    aggregator.assert_metric('consul.catalog.nodes_up', count=2)
+    aggregator.assert_metric('consul.catalog.total_nodes', count=2)
```

Comment:
Is adding `aggregator.assert_all_metrics_covered()` appropriate here ?

Reply:
I don't think it is, since I cannot understand why `consul.net.node.latency.*` metrics are not always here. Seems it caused some trouble already https://github.com/DataDog/integrations-core/blob/b863cb76863304b80cceeddf9b8b7bbd9c994294/consul/tests/test_integration.py#L21